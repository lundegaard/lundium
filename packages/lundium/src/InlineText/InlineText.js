import React from 'react';
import PropTypes from 'prop-types';

import Text from '../Text';

const InlineText = ({ as = 'span', ...rest }) => <Text as={as} {...rest} />;

InlineText.propTypes = {
	/** Tag used for root component */
	as: PropTypes.any,
	...Text.propTypes,
};

export default InlineText;
