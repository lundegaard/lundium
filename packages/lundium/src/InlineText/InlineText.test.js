import React from 'react';
import { shallow } from 'enzyme';

import InlineText from './InlineText';

describe('<InlineText />', () => {
	const requiredProps = {};

	it('shallow', () => {
		const wrapper = shallow(<InlineText {...requiredProps} />);
		expect(wrapper).toMatchSnapshot();
	});
});
