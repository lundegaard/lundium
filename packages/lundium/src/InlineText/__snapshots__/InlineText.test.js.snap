// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<InlineText /> shallow 1`] = `
<forwardRef(Text)
  as="span"
/>
`;
